src/Options.php

```diff
@@ -701,6 +701,28 @@ public function setMaxRequestBodySize(string $maxRequestBodySize): void
         $this->options = $this->resolver->resolve($options);
     }
 
+    /**
+     * Gets the type serializers.
+     *
+     * @return array
+     */
+    public function getClassSerializers(): array
+    {
+        return $this->options['class_serializers'];
+    }
+
+    /**
+     * Sets the class serializers.
+     *
+     * @param array $serializers
```

Comment:
Change the description to `The list of serializer callbacks` and the typehint to `array<string, callable>`